### ISSUE TITLE
Bundled `fmt` compilation failure (and fix)

### ISSUE THREAD

**Issue by harrism** [gabime/spdlog#1661]
Great library, thanks! (We are using spdlog in https://github.com/rapidsai/rmm and other RAPIDS framework libraries)

There is a compilation issue caused by `fmt` library on libstdc++ systems with C++ < 17. See https://github.com/fmtlib/fmt/issues/1850

The above is fixed by https://github.com/fmtlib/fmt/pull/1852, so please incorporate this fix once it is merged into `fmt`.

**Comment by gabime** [comment]
Thanks for the nice feedback! I am glad you find it useful.

Regarding the fix, spdlog only bundles official fmt versions, so it will have to wait for the next fmt release.

If this is showstopper for you, you can make spdlog to use your own fixed version of fmt by defining SPDLOG_FMT_EXTERNAL (or  -DSPDLOG_FMT_EXTERNAL=True if using CMake).

**Comment by harrism** [comment]
Sorry for the noise, it appears this is not an issue with `fmt`. Closing.
